tests/platform/cloud/AcceptanceTests_untrustedSettings.js

```diff
@@ -115,34 +124,50 @@ gpii.tests.cloud.untrustedSettings.testDefs = [
         name: "Anonymous token (token without a user account)",
         userToken: "os_gnome",
         isAnonymousToken: true,
-        expectedSettings: {
-            "org.gnome.desktop.a11y.magnifier": {
-                "settingsHandlers": {
-                    "configuration": {
-                        "settings": {
-                            "mag-factor": 1.5,
-                            "screen-position": "full-screen"
-                        }
-                    }
-                }
-            },
-            "org.gnome.desktop.interface": {
-                "settingsHandlers": {
-                    "configuration": {
-                        "settings": {
-                            "text-scaling-factor": 0.75,
-                            "cursor-size": 90,
-                            "gtk-theme": "Adwaita", // added by default if high contrast isn't on
-                            "icon-theme": "gnome" // added by default if high contrast isn't on
-                        }
-                    }
-                }
-            },
-            "org.alsa-project": {
-                "settingsHandlers": {
-                    "configuration": {
-                        "settings": {
-                            "masterVolume": 50
+        expectedMatchMakerOutput: {
+            "inferredConfiguration": {
+                "gpii-default": {
+                    "applications": {
+                        "org.gnome.desktop.a11y.magnifier": {
+                            "active": true,
+                            "settings": {
+                                "http://registry.gpii.net/common/volume": 0.5,
```

Comment:
This looks dodgy still - why is the desktop magnifier getting allocated the volume common term?